test-suite/js/src/test/require-jdk8/org/scalajs/testsuite/javalib/math/BigIntegerConvertJSTest.scala

```diff
@@ -1,19 +0,0 @@
-package org.scalajs.testsuite.javalib.math
-
-import java.math.BigInteger
-
-import org.scalajs.jasminetest.JasmineTest
-
-object BigIntegerConvertJSTest extends JasmineTest {
-  describe("BigIntegerConvertTest") {
-    // To test that it works with strict floats, do:
-    //   > set scalaJSSemantics in testSuite ~= { _.withStrictFloats(true) }
-    when("strict-floats").
-    it("testDoubleValueZero") {
```

Comment:
Oh, it seems that I messed up when porting the rest of the tests, it was a tests called `testFloatValueBug2482` that had the `strict float`. I will add it back.